### ISSUE TITLE
Add a route existence check

### ISSUE THREAD

**Issue by roni-estein** [tightenco/ziggy#255]
It might be nice to have build in existence checking for a route
### Expected Behavior
Right now I've just added a global mixin

```js
routeExists(name) {
    try {
        route(name)
        return true;
    } catch (Err) {
        return false;
    }
},
```

On occasion I want to check if a route exists before offering a to the user.Having a convenient way to check it is always a plus. I'm sure there is a better implementation internally.

**Comment by Acen** [comment]
Ziggy internally as part of the UrlBuilder function does a quick undefined check against `this.ziggy.namedRoutes[supplied_route_name]` or alternatively it should throw an `Error`.

> `throw new Error('Ziggy Error: route \'' + this.name + '\' is not found in the route list');`

**Comment by bakerkretzmar** [comment]
Can't believe I didn't realize but we already have this: `route().check()` was added in #248 and released in [v0.8.1](https://github.com/tightenco/ziggy/releases/tag/v0.8.1) (I think).

It's not documented though, so I'll add it to the Readme. Thanks!

**Comment by jaxramus** [comment]
`route().check('login')` works awesome! Now I don't have to pass a prop for `Route::has('login')` everywhere.

Thank you.
